next/backend/dtos/minio/upload-minio-file.dto.ts

```diff
@@ -1,6 +1,6 @@
 export interface UploadMinioFile {
   file: File
   originalName: string
-  errorMessage?: string
+  errorMessage?: string[]
```

Comment:
@AntonZezul This is not part of RJSF. This is error of file validation on client and also server. Please roll back to simple string and test upload if its still working like it should.

Reply:
Okey, my mistake